webpack/assets/javascripts/react_app/components/ComponentWrapper/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import componentRegistry from '../componentRegistry';
+
+const ComponentWrapper = (props) => {
```

Comment:
`index.js` meant  for redux containers (connected components), I prefer to use `componentWrapper.js` as a name 
Plus, shouldn't it be under `common` directory?